openedx/core/djangoapps/schedules/tasks.py

```diff
@@ -197,6 +205,47 @@ def make_message_type(self, day_offset):
         return message_types.CourseUpdate()
 
 
+class ScheduleCourseNextSectionUpdate(ScheduleMessageBaseTask):
+    enqueue_config_var = 'enqueue_course_update'
+    log_prefix = COURSE_NEXT_SECTION_UPDATE_LOG_PREFIX
+    resolver = resolvers.CourseNextSectionUpdate
+    async_send_task = _course_update_schedule_send
+
+    @classmethod
+    def enqueue(cls, site, current_date, day_offset, override_recipient_email=None):
+        yesterday = (current_date - datetime.timedelta(days=day_offset)).date()
```

Comment:
Feels weird to label this variable `yesterday` when you're allowing a day_offset to be specified. Either change the variable name to something more general or just hardcode the day_offset as 1 and remove the function parameter.

You use Target Date language lower, that could be a good variable name.